From 2802871cbc11c4c1846cd3b3181229360ed02ca8 Mon Sep 17 00:00:00 2001
From: Nicola Corna <nicola@corna.info>
Date: Wed, 4 May 2022 07:30:32 +0200
Subject: [PATCH] Replace "username" with "e-mail" in the password dialogs

---
 src/password.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/password.c b/src/password.c
index a5490d3..1b7974d 100644
--- a/src/password.c
+++ b/src/password.c
@@ -70,7 +70,7 @@ zenity_password_dialog (ZenityData *data, ZenityPasswordData *password_data) {
 	gchar *title_text = _ ("Type your password");
 
 	if (password_data->username)
-		title_text = _ ("Type your username and password");
+		title_text = _ ("Type your e-mail and password");
 
 	label = gtk_label_new (title_text);
 
@@ -97,7 +97,7 @@ zenity_password_dialog (ZenityData *data, ZenityPasswordData *password_data) {
 	gtk_box_pack_start (GTK_BOX (hbox), vbox_entries, TRUE, TRUE, 12);
 
 	if (password_data->username) {
-		label = gtk_label_new (_ ("Username:"));
+		label = gtk_label_new (_ ("E-mail:"));
 		gtk_box_pack_start (GTK_BOX (vbox_labels), label, TRUE, FALSE, 12);
 		password_data->entry_username = gtk_entry_new ();
 		gtk_box_pack_start (GTK_BOX (vbox_entries),
-- 
2.36.0

